.github/workflows/inviwo.yml

```diff
@@ -77,8 +82,15 @@ jobs:
       with:
         repository: 'microsoft/vcpkg'
         path: 'vcpkg' # The cmake vcpkg present expects vcpkg to be next to the inviwo source folder
-    
+
+    - name: 'Bootstrap vcpkg'
+      shell: bash
+      run: |
+        ${{ matrix.bootstrap_vcpkg }} 
+        ./vcpkg/vcpkg integrate install
+
     # Cache built dependencies for faster subsequent builds
+    # ./vcpkg/bootstrap-vcpkg.bat must have been called before
```

Comment:
Since when?

Reply:
What is not working for Mac/linux? The caching?

Reply:
Yes, exactly, it fails when trying to upload. But I think that I have fixed it now (see change for nuget setup). Unsure if the bootstrapping before is necessary.

for some reason it does not work for the visualneuro repo. Perhaps the repo  is used in the nuget setup somehow